test/stripe/api_resource_test.rb

```diff
@@ -160,7 +160,7 @@ class NestedTestAPIResource < Stripe::APIResource
           .with(query: { customer: "cus_123" })
           .to_return(body: JSON.generate(data: [charge_fixture],
                                          url: "/v1/charges"))
-        charges = Stripe::Invoice.list(customer: "cus_123")
+        charges = Stripe::Charge.list(customer: "cus_123")
```

Comment:
Judging by the parameters sent and the variable naming, this was supposed to be `Charge.list` all along. I only noticed the problem because the invoice list endpoint doesn't take a `created` parameter.